docs/topics/Gateway.md

```diff
@@ -181,7 +181,7 @@ Used to tell clients to reconnect to the gateway. Clients should immediately rec
 
 ### Gateway Request Guild Members
 
-Used to request offline members for a guild. When initially connecting, the gateway will only send offline members if a guild has less than the `large_threshold` members (value in the [Gateway Identify](#DOCS_GATEWAY/gateway-identify)). If a client wishes to receive all members, they need to explicitly request them. The server will send a [Guild Members Chunk](#DOCS_GATEWAY/guild-members-chunk) event in response.
+Used to request offline members for a guild. When initially connecting, the gateway will only send offline members if a guild has less than the `large_threshold` members (value in the [Gateway Identify](#DOCS_GATEWAY/gateway-identify)). If a client wishes to receive additional members, they need to explicitly request them. The server will send [Guild Members Chunk](#DOCS_GATEWAY/guild-members-chunk) events in response with up to 1000 members per chunk.
```

Comment:
This could even be a bit clearer. I think it should say `Guild Members Chunk events`, plural, to denote multiple events can fire. And perhaps clarify afterwards with something like:

`These events will fire until all members have been sent, or the specified limit has been reached.`